pkg/test/resourcefixture/testdata/resourceoverrides/dnsrecordset/routingpoliciesgeohealthchecks/create.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSRecordSet
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: ${projectId}
+  name: dnsrecordset-${uniqueId}
+spec:
+  name: "cnrm-dns-example-${uniqueId}.com."
+  type: A
+  ttl: 300
+  managedZoneRef:
+    name: dnsmanagedzone-${uniqueId}
+  projectRef:
```

Comment:
Remove? This is not a valid field in DNSRecordSet: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_dnsrecordsets.dns.cnrm.cloud.google.com.yaml